Source/Public/Update-VSTeamUserEntitlement.ps1

```diff
@@ -69,7 +65,7 @@ function Update-VSTeamUserEntitlement {
          }
       }
 
-      $body = ConvertTo-Json -InputObject @($obj) -Depth 100
+      $body = ConvertTo-Json -InputObject @($obj) -Depth 100 -Compress
```

Comment:
I can't put this comment down below by the write-outputs, but I'd consider Write-Information or Write-Host or create a structured object to hold the new and old values.  Write-output puts that text on the pipeline and may lead to unexpected behavior in a pipeline.  Another option would be a `-passthru` parameter to pass the updated object along the pipeline.

Probably not a change for now, but worth considering for future releases.

Reply:
changed this now to Write-Information